### PR TITLE
my homework of 07082023

### PR DESCRIPTION
**Pull request by sandeepsainihisar**

### CODE REVIEW

File: js_july_23/Users/Sandeep/Homework/homework Variable07082023.html
Comment (manita1993) [inline js_july_23/Users/Sandeep/Homework/homework Variable07082023.html:1]: At line no. 19 why you added "toFixed"?

Can you clarify it?

**Review by manita1993 — COMMENTED** [review]
Added comments

**Review by manita1993 — APPROVED** [review]
Added comments

File: js_july_23/Users/Sandeep/Homework/homework Variable07082023.html
Comment (sandeepsainihisar) [reply js_july_23/Users/Sandeep/Homework/homework Variable07082023.html:1]: tofixed is an predefined function of js to roundoff the how many number will see the after the decimal

**Pull request merged**